pyvista/examples/downloads.py

```diff
@@ -3706,3 +3707,59 @@ def download_cgns_multi(load=True):  # pragma: no cover
     if not load:
         return filename
     return pyvista.get_reader(filename).read()
+
+
+def download_dicom_stack(load: bool = True) -> Union[pyvista.UniformGrid, str]:  # pragma: no cover
+    """Download TCIA DICOM stack volume.
+
+    Original download from the `The Cancer Imaging Archive (TCIA)
+    <https://www.cancerimagingarchive.net/>`_. This is part of the
+    Clinical Proteomic Tumor Analysis Consortium Sarcomas (CPTAC-SAR)
+    collection.
+
+    Parameters
+    ==========
+    load : bool, optional
+        Load the dataset after downloading it when ``True``.  Set this
+        to ``False`` and only the filename will be returned.
+
+    Returns
+    =======
+    pyvista.UniformGrid or str
+        DataSet or filename depending on ``load``.
+
+    References
+    ==========
+
+        Data Citation
+        -------------
+
+        National Cancer Institute Clinical Proteomic Tumor Analysis Consortium (CPTAC). (2018).
+        Radiology Data from the Clinical Proteomic Tumor Analysis Consortium Sarcomas
+        [CPTAC-SAR] collection [Data set]. The Cancer Imaging Archive.
+        DOI: 10.7937/TCIA.2019.9bt23r95
+
+        Acknowledgement
+        ---------------
+
+        Data used in this publication were generated by the National Cancer Institute Clinical
+        Proteomic Tumor Analysis Consortium (CPTAC).
+
+        TCIA Citation
+        -------------
+
+        Clark K, Vendt B, Smith K, Freymann J, Kirby J, Koppel P, Moore S, Phillips S,
+        Maffitt D, Pringle M, Tarbox L, Prior F. The Cancer Imaging Archive (TCIA):
+        Maintaining and Operating a Public Information Repository, Journal of Digital Imaging,
+        Volume 26, Number 6, December, 2013, pp 1045-1057. doi: 10.1007/s10278-013-9622-7
+
+    Examples
+    ========
+    >>> from pyvista import examples
+    >>> dataset = examples.download_dicom_stack()
+    >>>
```

Comment:
> It would be nice to plot a slice like your image from the PR here.

I can do that:

```python
>>> from pyvista import examples
>>> volume = examples.download_dicom_stack()
>>> volume.plot()
```